<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TosaveDialog</class>
 <widget class="QDialog" name="TosaveDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(201,101,0);</string>
  </property>
  <widget class="QRadioButton" name="radioButton_same">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>     save in the same file</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_different">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>     save in different file</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_save">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>please enter the name file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>130</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:7px;
border:1px solid grey;
background:teal;
color:white</string>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
